owncloudApp/src/androidTest/java/com/owncloud/android/files/details/FileDetailsFragmentTest.kt

```diff
@@ -0,0 +1,214 @@
+import com.owncloud.android.presentation.files.operations.FileOperationsViewModel
+import com.owncloud.android.sharing.shares.ui.TestShareFileActivity
+import com.owncloud.android.testutil.OC_ACCOUNT
+import com.owncloud.android.testutil.OC_FILE
+import com.owncloud.android.testutil.OC_FILE_OC_AVAILABLE_OFFLINE_FILE
+import com.owncloud.android.testutil.OC_FILE_WITH_SYNC_INFO
+import com.owncloud.android.testutil.OC_FILE_WITH_SYNC_INFO_AND_SPACE
+import com.owncloud.android.testutil.OC_FILE_WITH_SYNC_INFO_AND_WITHOUT_PERSONAL_SPACE
+import com.owncloud.android.utils.DisplayUtils
+import com.owncloud.android.utils.matchers.assertVisibility
+import com.owncloud.android.utils.matchers.isDisplayed
+import com.owncloud.android.utils.matchers.withDrawable
+import com.owncloud.android.utils.matchers.withText
+import io.mockk.every
+import io.mockk.mockk
+import kotlinx.coroutines.flow.MutableStateFlow
+import org.junit.Before
+import org.junit.Test
+import org.koin.androidx.viewmodel.dsl.viewModel
+import org.koin.core.context.startKoin
+import org.koin.core.context.stopKoin
+import org.koin.dsl.module
+
+class FileDetailsFragmentTest {
+
+    private lateinit var fileDetailsViewModel: FileDetailsViewModel
+    private lateinit var fileOperationsViewModel: FileOperationsViewModel
+    private lateinit var context: Context
+
+    private var currentFile: MutableStateFlow<OCFileWithSyncInfo?> = MutableStateFlow(OC_FILE_WITH_SYNC_INFO_AND_SPACE)
+    private var currentFileWithoutPersonalSpace: MutableStateFlow<OCFileWithSyncInfo?> =
+        MutableStateFlow(OC_FILE_WITH_SYNC_INFO_AND_WITHOUT_PERSONAL_SPACE)
+    private var currentFileSyncInfo: MutableStateFlow<OCFileWithSyncInfo?> = MutableStateFlow(OC_FILE_WITH_SYNC_INFO)
+    private var currentFileAvailableOffline: MutableStateFlow<OCFileWithSyncInfo?> = MutableStateFlow(OC_FILE_OC_AVAILABLE_OFFLINE_FILE)
+
+    @Before
+    fun setUp() {
+        context = ApplicationProvider.getApplicationContext()
+        fileDetailsViewModel = mockk(relaxed = true)
+        fileOperationsViewModel = mockk(relaxed = true)
+        every { fileDetailsViewModel.currentFile } returns currentFile
+        stopKoin()
+
+        startKoin {
+            context
+            allowOverride(override = true)
+            modules(
+                module {
+                    viewModel {
+                        fileDetailsViewModel
+                    }
+                    viewModel {
+                        fileOperationsViewModel
+                    }
+                }
+            )
+
+        }
+
+        val fileDetailsFragment = FileDetailsFragment.newInstance(
+            OC_FILE,
+            OC_ACCOUNT,
+            syncFileAtOpen = false
+        )
+
+        launch(TestShareFileActivity::class.java).onActivity {
+            it.startFragment(fileDetailsFragment)
+        }
+    }
+
+    @Test
+    fun initial_status_components() {
```

Comment:
In Android projects, identifiers (such as method names, variables, classes, etc.) cannot contain spaces.

The convention of using backticks and spaces in test method names is a Kotlin language-specific feature, but it can cause problems in Android projects, since Android follows stricter naming rules compared to Kotlin in this aspect. 

It is working in test directory, but not in androidTest.